### PR TITLE
Rename "trash bin" to the "deleted files".

### PR DESCRIPTION
**Pull request by PVince81**
## Related Issue
Part 5 of https://github.com/owncloud/product/issues/231

## How Has This Been Tested?
Manual test, visually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: apps/files/src/components/FilesAppBar.vue [hunk 1]
```diff
@@ -127,7 +127,7 @@
               selectedFiles.length < 1 ? $_ocTrashbin_empty() : $_deleteResources_displayDialog()
             "
           >
-            {{ $_ocAppBar_clearTrashbinButtonText }}
+            <translate>Delete</translate>
```
Comment (LukasHirt) [inline apps/files/src/components/FilesAppBar.vue:130]: I'm a bit worried that having only "Delete" won't be clear enough when no files are selected. Could we keep the separation and use something like "Delete all files"? Or maybe just "Empty" (this is how mac os shows it in the bin)?

Comment (PVince81) [reply apps/files/src/components/FilesAppBar.vue:130]: ah, I didn't realize that the button was clickable even with zero selection

I'll rename it to "Empty" in that case

Comment (LukasHirt) [reply apps/files/src/components/FilesAppBar.vue:130]: Great, thanks!

**Review by LukasHirt — APPROVED** [review]
Feel free to merge once the text in button is changed :wink:

File: apps/files/src/components/FilesAppBar.vue (hunk 1 above)
Comment (PVince81) [reply apps/files/src/components/FilesAppBar.vue]: done

**Comment by PVince81** [comment]
fixed the test, rebased and squashed

**Comment by PVince81** [comment]
missed one test :facepalm:

**Comment by PVince81** [comment]
okay, so that test is skipped on OC 10 that's why I didn't see it in my local run.

fixed now

**Comment by PVince81** [comment]
some bad luck about time outs: https://drone.owncloud.com/owncloud/phoenix/11021/36/12

that test passed before... restarted...

**Comment by PVince81** [comment]
I also have the feeling that Drone is running slower. During the afternoon most test runs were able to finish in max 30+ minutes but now they take even longer.

**Comment by phil-davis** [comment]
dependabot added or auto-rebased a bunch of PRs. drone got a queue of jobs.

**Pull request merged**